docs/src/pages/system/box/box.md

```diff
@@ -0,0 +1,76 @@
+---
+title: React Box component
+githubLabel: 'component: Box'
```

Comment:
too?

```suggestion
components: Box
githubLabel: 'component: Box'
```

Reply:
reverted in both pages. The component has a custom API in the end of the page.